### PR TITLE
Issue 616 - Add Chrome results for media playback tests

### PR DESCRIPTION
**Pull request by airimovici**
I added the Chrome tests for desktop.

The summary page is a lot slower now @klahnakoski

### CODE REVIEW

**Comment by airimovici** [comment]
Issue https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues/616

File: src/playback/summary.jsx [hunk 1]
```diff
@@ -276,6 +276,7 @@ class PlaybackSummary extends React.Component {
                   </div>
                 </Grid>
                 {selectFrom(SPECIAL_SIZES).map(({ id, label }) => {
+                  console.log(browserId, 'scores', scores);
```
Comment (klahnakoski) [inline src/playback/summary.jsx]: remove this log line

File: src/playback/summary.jsx [hunk 2]
```diff
@@ -200,7 +200,7 @@ class PlaybackSummary extends React.Component {
     const {
       bits, encoding, classes, browserId,
     } = this.props;
-    const platformType = browserId === 'firefox' ? 'desktop' : 'mobile';
+    const platformType = browserId === 'firefox' || browserId === 'chrome' ? 'desktop' : 'mobile';
```
Comment (klahnakoski) [inline src/playback/summary.jsx]: This logic is a little messy.  We should have this desktop/mobile distinction in our metadata somewhere.

No need to change it, but keep it in mind for the future

File: src/playback/config.js [hunk 3]
```diff
@@ -123,6 +123,7 @@ const PLATFORMS = [
 
 const BROWSERS =[
   {id: "firefox", label:"Firefox", title:"Firefox (Desktop)", filter: {eq: {framework: 10, repo: 'mozilla-central', suite: ["raptor-youtube-playback-firefox-live"]}}},
+  {id: "chrome", label:"Chrome", title:"Chrome (Desktop)", filter: {eq: {framework: 10, repo: 'mozilla-central', suite: ["raptor-youtube-playback-chrome-live"]}}},
```
Comment (klahnakoski) [inline src/playback/config.js]: ...yes:  We should be adding a property to these records, like `{format: "desktop"}`, and using that.

File: src/playback/summary.jsx (hunk 1 above)
Comment (airimovici) [reply src/playback/summary.jsx]: thanks for that, I forgot about it :)

**Comment by davehunt** [comment]
@klahnakoski can this be merged?

**Comment by airimovici** [comment]
We should be able to merge this

**Pull request merged**